### DISCUSSION TITLE
What does This can be speed this up by passing a ElevenLabsConfiguration to the ElevenLabsAuthentication.ctr mean?

### DISCUSSION THREAD

**Discussion by tomkail** [RageAgainstThePixel/com.rest.elevenlabs#69] [Q&A] [answered]
Hi! We're finding this log spams the console, but we don't know what it means:
`This can be speed this up by passing a ElevenLabsConfiguration to the ElevenLabsAuthentication.ctr`
We don't know what/where the ElevenLabsAuthentication.ctr file lives. Can you help?
Ta!

**Comment by StephenHodgson** [comment]
This means the loading and start time can be optimized if you cache a reference to your configuration and pass it through the constructor of your client.

See updated sample scene on how to do this.
https://github.com/RageAgainstThePixel/com.rest.elevenlabs/blob/5852e47f625d93a0e9d75bce9fdcf6502a5c7f33/ElevenLabs/Packages/com.rest.elevenlabs/Samples~/TextToSpeech/TextToSpeechDemo.cs#L50

**Reply by StephenHodgson** [reply, marked as answer]
Also see https://github.com/RageAgainstThePixel/com.rest.elevenlabs/discussions/67
